### DISCUSSION TITLE
SystemAppData\Helium contents

### DISCUSSION THREAD

**Discussion by vertigo220** [microsoft/terminal#13750] [Q&A] [answered]
This directory contains the files User.dat and UserClasses.dat, as well as what I assume are backups of those files with .LOG1 and .LOG2 appended to their names. My questions is, what are these files, i.e. what data is stored in them? I ask because I like to make backups of the settings for my apps, which allows me to restore them after reinstalling or onto a different computer, but robocopy fails on these files with

```
ERROR 2 (0x00000002) Changing File Attributes C:\Users\Username\AppData\Local\Packages\Microsoft.WindowsTerminal_8wekyb3d8bbwe\SystemAppData\Helium\UserClasses.dat.LOG2
The system cannot find the file specified.
```

and trying with the /B parameter, as I've found suggested elsewhere, doesn't work either (and it actually says "ERROR 0 (0x00000000)..." followed immediately by "The operation completed successfully".../smh), nor does /COPY:DT.

So I need to know if it's worth trying to figure out what MS stupidity is interfering to get it working, or just find another way entirely, or if these files can simply be safely excluded without losing any configuration I may want later.

**Comment by DHowett** [comment, marked as answer]
They're registry hives! You should be able to mount them using regedit.

<img width="222" alt="image" src="https://user-images.githubusercontent.com/189190/184730505-5062d34c-b74f-4d42-9996-74c1ae00865f.png">

<img width="844" alt="image" src="https://user-images.githubusercontent.com/189190/184730242-36395917-e668-430b-909d-3e87083c0bed.png">

<img width="301" alt="image" src="https://user-images.githubusercontent.com/189190/184730313-a829ac56-630b-491d-b024-47064ba2c03d.png">

<img width="215" alt="image" src="https://user-images.githubusercontent.com/189190/184730424-3c0bbc72-5508-4cfc-b8f8-68402ddeb056.png">

We're explicitly not storing any user settings in the registry, so you can ignore them for the purposes of archiving your preferences. :smile:

**Reply by DHowett** [reply]
The LOG files are part of how the Windows Registry works; they're used for storing incomplete transactions ala a "transaction log".

**Reply by vertigo220** [reply]
Thanks!
